src/features/todos.ts

```diff
@@ -1,9 +1,32 @@
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type SetAddAction = {
+  type: 'todos/SET_ADD',
+  playload: Todo[],
+};
+
+const SetAdd = (arr: Todo[]): SetAddAction => ({
+  type: 'todos/SET_ADD',
+  playload: arr,
+});
+
+export const actions = { SetAdd };
+
+const initialTodos: Todo[] = [];
+
+type Action = SetAddAction;
+
+const todosReducer = (
+  state = initialTodos,
+  action: Action,
+): Todo[] => {
+  switch (action.type) {
+    case 'todos/SET_ADD':
+      return [...action.playload, ...state];
```

Comment:
This line may cause a problem because it adds the new todos to the front of the array. If you want to add new todos to the end of the array, you should swap the array elements in the return statement to return [...state, ...action.payload]